package.json

```diff
@@ -179,7 +182,8 @@
     "rollup-plugin-styles": "^4.0.0",
     "semver": "^7.3.8",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
+    "typescript": "^4.9.5",    
+    "vite": "~4.1.4",
```

Comment:
I'm not really comfortable having unrelated dependencies here, I really do think everything related to the demo shouldn't impact the main project (this file is even published)

Reply:
Ok, I wouldn't be this strict here, but I understand.

I added `monaco-editor` as a direct dependency to **demo** again and it works (codicons are used). There is an error in the console, but it works due to the direct dependency being used in the end. 
I hope this ok now. 🙂